### DISCUSSION TITLE
What does Manifold::Minus actually buy you?

### DISCUSSION THREAD

**Discussion by pjozog** [ceres-solver/ceres-solver#908] [General]
Hello, just wondering what the new `Minus` operator buys you in a parameter block. I have no experience with needing these in a least squares solver (besides I guess them being handy in computing user-provided resiudals). @sandwichmaker says something will happen in 2.2, but what?

Thanks.

**Comment by sandwichmaker** [comment]
@pjozog I have not had time to work on this, but the Minus operator allows to compute distances in the tangent space, which means for example 

The following computation

https://github.com/ceres-solver/ceres-solver/blob/master/internal/ceres/trust_region_minimizer.cc#L274

can be done correctly. Minus is also needed (though this is a topic of discussion right now, whether our current interface is enough or we need something more general) for implementing Marginalization in a generic fashion.
